views/mail/unban.htm

```diff
@@ -0,0 +1,12 @@
+subject = "Account {{ name }} {{ surname }} has been unbanned."
+description = "An admin has unbanned an account"
+==
+The following user's account has been unbanned.
+
+This normally happens when an admin manually removes a ban on them:
+
+* Name: {{ name }}
+* Surname: {{ surname }}
+* Email: {{ email }}
+
+User Management: [{{ link }}]({{ link }})
```

Comment:
Should this include the current backend user making the request to unban the user if there is one?

Reply:
That would be helpful to know who is responsible for the action, yes

Reply:
Are you able to add that to the PR?